docs/UsersGuide/source/ConfigParameters.inc

```diff
@@ -233,106 +668,347 @@ Set parameters associated with the fixed (i.e. static) files.  For the main NOAA
+
+
+Stochastic Physics Parameters
+================================
+
+..
+   COMMENT: Move all physic params up with CCPP section?
+
+For the most updated and detailed documentation of these parameters, see the `UFS Stochastic Physics Documentation <https://stochastic-physics.readthedocs.io/en/latest/namelist_options.html>`__.
+
+
+``NEW_LSCALE``: (Default: "TRUE") 
+   #
+
+..
+   COMMENT: Cannot find information on NEW_LSCALE variable in the stochastic physics docs. 
+
+Specific Humidity (SHUM) Perturbation Parameters
+---------------------------------------------------
+
+``DO_SHUM``: (Default: "FALSE")
+   Flag to turn Specific Humidity (SHUM) perturbations on or off. SHUM perturbations multiply the low-level specific humidity by a small random number at each time-step. The SHUM scheme attempts to address missing physics phenomena (e.g., cold pools, gust fronts) most active in convective regions. 
+
+``ISEED_SHUM``: (Default: "2")
+   Seed for setting the SHUM random number sequence.
+
+``SHUM_MAG``: (Default: "0.006") 
+   Amplitudes of random patterns. Corresponds to the variable ``shum`` in ``input.nml``.
+
+``SHUM_LSCALE``: (Default: "150000")
+   Decorrelation spatial scale in meters.
+
+``SHUM_TSCALE``: (Default: "21600")
+   Decorrelation timescale in seconds. Corresponds to the variable ``shum_tau`` in ``input.nml``.
+
+``SHUM_INT``: (Default: "3600")
+   Interval in seconds to update random pattern (optional). Perturbations still get applied at every time-step. Corresponds to the variable ``shumint`` in ``input.nml``.
+
+.. _SPPT:
+
+Stochastically Perturbed Physics Tendencies (SPPT) Parameters
+-----------------------------------------------------------------
+
+SPPT perturbs full physics tendencies *after* the call to the physics suite, unlike `SPP <SPP>` (below), which acts on the physics itself. 
+
+``DO_SPPT``: (Default: "FALSE")
+   Flag to turn Stochastically Perturbed Physics Tendencies (SPPT) on or off. SPPT multiplies the physics tendencies by a random number O [0,2] before updating the model state. Addresses error in the physics parameterizations (either missing physics or unresolved subgrid processes). It is most active in the boundary layer and convective regions. 
+
+..
+   COMMENT: Does it need to be O? can it be X? or some other variable/letter that doesn't look like the number zero? 
```

Comment:
This is confusion, good catch.  I suggest changing it to:
SPPT multiplies the physics tendencies by a random number between 0 and 2 before updating the model state.